lib/lita/handlers/totems.rb

```diff
@@ -212,6 +231,10 @@ def info(response)
         response.reply resp
       end
 
+      def stats(response)
+        response.reply %{https://app.signalfx.com/#/dashboard/FNSkEUVAYAA}
```

Comment:
I wonder if this should be made a constant and referenced here instead at least for code comprehension. 
ie. 

```ruby
TOTEMS_DASHBOARD = "https://app.signalfx.com/#/dashboard/FNSkEUVAYAA"

...

def stats(response) response.reply %{TOTEMS_DASHBOARD} end
```